test/dynamo/test_flash_attention.py

```diff
@@ -0,0 +1,43 @@
+# Owner(s): ["module: dynamo"]
+import contextlib
+
+import torch._dynamo.test_case
+import torch._dynamo.testing
+from torch.backends.cuda import SDPAParams
+
+
+@contextlib.contextmanager
+def allow_in_graph_sdpa_params():
+    global SDPAParams
+    try:
+        old = SDPAParams
+        SDPAParams = torch._dynamo.allow_in_graph(SDPAParams)
+        yield
+    finally:
+        SDPAParams = old
+
+
+class TestFlashAttention(torch._dynamo.test_case.TestCase):
+    def test_returns_SDPAParams(self):
+        with allow_in_graph_sdpa_params():
+
+            @torch.compile()
```

Comment:
Test with compile counter, this whole thing could be graph breaking for all I know, and the test would pass!

Reply:
I'll just do fullgraph